src/Home/Home.tsx

```diff
@@ -130,6 +130,8 @@ const HomeScreen: FunctionComponent = () => {
     navigation.navigate(Screens.LanguageSelection)
   }
 
+  const showShareLink = configuration.appDownloadLink !== null
```

Comment:
This is making a transformation on a value we set up, how do you feel about setting this setting higher? Or perhaps using the null value as a boolean with the casting `Boolean(appDownloadLink)` down below? It's probably fine but I'm worried that we keep pushing defensive code down the pipe when we could avoid it by setting nondisruptive values on the context(this null seems nondisruptive to me), maybe it's just because this screen is getting a little large and we could move the share link to a component that renders null if no download link is present? ... Sorry for the rant, I hope it makes sense 😬

Reply:
Makes a lot of sense! I agree that the explicit null checking is not the best pattern, so will make an update. thanks!